src/style.css

```
@@ -1,3 +1,70 @@
-body {
+:root { --main-text-color: #00acdc;
+}
+
```

Don't forget about code style
```suggestion
:root { 
  --main-text-color: #00acdc;
}

```